keyboards/4pplet/eagle_viper_rep/rev_a/readme.md

```diff
@@ -0,0 +1,12 @@
+# Viper/Eagle REP Rev A
+
+Replacement PCB for Viper/Eagle V2/V3
+
+* Keyboard Maintainer: [4pplet](https://github.com/4pplet)
+* Hardware Supported: Viper/Eagle REP Rev A
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware?

Reply:
Instruction on how to enter DFU added 👍